tailwind/Link.tsx

```diff
@@ -0,0 +1,193 @@
+import { AnchorHTMLAttributes, forwardRef } from "react"
+import NextLink, { type LinkProps as NextLinkProps } from "next/link"
+import { useRouter } from "next/router"
+import { RxExternalLink } from "react-icons/rx"
+import { Icon } from "@chakra-ui/react"
+import { VisuallyHidden } from "@radix-ui/react-visually-hidden"
+
+import { cn } from "@/lib/utils/cn"
+import { type MatomoEventOptions, trackCustomEvent } from "@/lib/utils/matomo"
+import { getRelativePath } from "@/lib/utils/relativePath"
+import * as url from "@/lib/utils/url"
+
+import { DISCORD_PATH, SITE_URL } from "@/lib/constants"
+
+import { useRtlFlip } from "@/hooks/useRtlFlip"
+
+type BaseProps = {
+  /** @deprecated Use `href` prop instead */
+  to?: string
```

Comment:
At this point, could we officially remove the `to` prop here? We've been migrating these over time, haven't checked if there are any usages left

Reply:
Not yet, we still have many places where we are using this prop. However, should be a pretty easy refactor but wouldn't like to include that in this PR just to keep it small.

Reply:
Marking this as resolved. I'll create an issue to keep track of this refactor.

Reply:
#13423